x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/search_cases.ts

```diff
@@ -0,0 +1,809 @@
+              ],
+            },
+          })
+        );
+
+        await createConfiguration(
+          supertest,
+          getConfigurationRequest({
+            overrides: {
+              customFields: [
+                {
+                  key: 'valid_obs_key_1',
+                  label: 'text',
+                  type: CustomFieldTypes.TEXT,
+                  required: false,
+                },
+                {
+                  key: 'valid_obs_key_2',
+                  label: 'toggle',
+                  type: CustomFieldTypes.TOGGLE,
+                  required: true,
+                },
+                {
+                  key: 'valid_obs_key_3',
+                  label: 'another_toggle',
+                  type: CustomFieldTypes.TOGGLE,
+                  required: false,
+                },
+              ],
+              owner: 'observability',
+            },
+          })
+        );
+
+        const secCase = await createCase(supertest, {
+          ...postCaseReq,
+          owner: 'securitySolutionFixture',
+          customFields: [
+            {
+              key: 'valid_key_1',
+              type: CustomFieldTypes.TEXT,
+              value: null,
+            },
+            {
+              key: 'valid_key_2',
+              type: CustomFieldTypes.TOGGLE,
+              value: false,
+            },
+            {
+              key: 'valid_key_3',
+              type: CustomFieldTypes.TOGGLE,
+              value: true,
+            },
+          ],
+        });
+
+        const obsCase = await createCase(supertest, {
+          ...postCaseReq,
+          owner: 'observability',
+          customFields: [
+            {
+              key: 'valid_obs_key_1',
+              type: CustomFieldTypes.TEXT,
+              value: null,
+            },
+            {
+              key: 'valid_obs_key_2',
+              type: CustomFieldTypes.TOGGLE,
+              value: false,
+            },
+            {
+              key: 'valid_obs_key_3',
+              type: CustomFieldTypes.TOGGLE,
+              value: true,
+            },
+          ],
+        });
+
+        expect(
+          await searchCases({
+            supertest,
+            body: { customFields: { valid_key_2: [false] } },
+          })
+        ).to.eql({
+          ...findCasesResp,
+          total: 1,
+          cases: [secCase],
+          count_open_cases: 1,
+        });
+
+        expect(
+          await searchCases({
+            supertest,
+            body: { customFields: { valid_obs_key_2: [false] } },
+          })
+        ).to.eql({
+          ...findCasesResp,
+          total: 1,
+          cases: [obsCase],
+          count_open_cases: 1,
+        });
+
+        expect(
+          await searchCases({
+            supertest,
+            body: { customFields: { valid_key_2: [false], valid_obs_key_2: [false] } },
+          })
+        ).to.eql({
+          ...findCasesResp,
+          total: 0,
+          cases: [],
```

Comment:
No, it should be no cases because here `body: { customFields: { valid_key_2: [false], valid_obs_key_2: [false] } },`  it is trying to search for a case which has `valid_key_2: [false] AND valid_obs_key_2: [false]`. 

No case has a customField from `observability and securitySolution`

Reply:
Thanks for the explanation! Make sense. I was confused 🙂.